charms/openfga-k8s/tox.ini

```diff
@@ -61,7 +61,7 @@ deps =
     -r{toxinidir}/requirements.txt
 commands =
     coverage run --source={[vars]src_path} \
-        -m pytest --ignore={[vars]tst_path}integration -v --tb native -s {posargs}
+        -m pytest --ignore={[vars]tst_path}integration -vv --tb native -s {posargs}
```

Comment:
This `-vv` is different from the integration test environment below (there it's `-v`). Is it intentional?

Reply:
reverted